POEApi.Model/Gem.cs

```diff
@@ -6,13 +6,41 @@ namespace POEApi.Model
     public class Gem : SocketableItem
     {
         public int Level { get; set; }
+        public double LevelProgress { get; private set; }
+        public int Numerator { get; set; }
+        public int Denominator { get; set; }
 
         public Gem(JSONProxy.Item item) : base(item)
         {
             this.Properties = ProxyMapper.GetProperties(item.Properties);
             this.ItemType = ItemType.Gem;
 
             this.Level = getLevel();
+
+            ExtractGemExperience(item);
+        }
+
+        private void ExtractGemExperience(JSONProxy.Item item)
+        {
+            var experienceProperties = item.AdditionalProperties.FirstOrDefault(x => x.Name == "Experience");
+
+            if (experienceProperties == null)
+            {
+                return;
+            }
+
+            LevelProgress = experienceProperties.Progress;
+            var experienceValues = experienceProperties.Values[0].FirstOrDefault()?.ToString();
+
+            if (experienceValues == null || !experienceValues.Contains("/"))
+            {
+                return;
+            }
+
+            var numeratorAndDenominator = experienceValues.Split('/');
+
+            Numerator = int.Parse(numeratorAndDenominator[0]);
```

Comment:
Perhaps use something like `int.TryParse`?  If we're already assuming `numeratorAndDenominator` might be null or not have a `/` character, we might as well assume it might not have valid numbers.  I assume not, but just to double check: is it possible for the numerator to not be a whole number?